dto/scenario_iterations.go

```diff
@@ -31,19 +31,22 @@ type ScenarioIterationBodyDto struct {
 	Rules                         []RuleDto `json:"rules"`
 	ScoreReviewThreshold_deprec   *int      `json:"scoreReviewThreshold"` //nolint:tagliatelle
 	ScoreReviewThreshold          *int      `json:"score_review_threshold"`
-	ScoreRejectThreshold_deprec   *int      `json:"scoreRejectThreshold"` //nolint:tagliatelle
-	ScoreRejectThreshold          *int      `json:"score_reject_threshold"`
-	Schedule                      string    `json:"schedule"`
+	ScoreBlockAndReviewThreshold  *int      `json:"score_block_and_review_threshold"`
+	ScoreRejectThreshold_deprec   *int      `json:"scoreRejectThreshold"`   //nolint:tagliatelle
+	ScoreDeclineThreshold         *int      `json:"score_reject_threshold"` //nolint:tagliatelle
```

Comment:
`Decline` is changed but only for the "model" layer (= not in DTO and in DB), should we plan to propagate the change everywhere ?

Reply:
well actually it's already "decline" everywhere - it's the model layer that was out of sync. (though I missed a field to rename in the dbmodel package as well, I'll do that)